skrub/_table_vectorizer.py

```diff
@@ -557,16 +529,11 @@ def _more_tags(self) -> dict:
             "allow_nan": [True],
             "_xfail_checks": {
                 "check_complex_data": "Passthrough complex columns as-is.",
-                "check_dont_overwrite_parameters": (
-                    "`transformers` is modified during fit but it is not pass by the"
-                    " user. We need to create an instance of ColumnTransformer instead"
-                    " but the current behaviour is not leading to a bug."
-                ),
             },
         }
 
     @property
-    def is_parallelized(self) -> bool:
+    def _is_parallelized(self) -> bool:
```

Comment:
It seems the result would be incorrect if the user used the `parallel_config` or `parallel_backend` from joblib? in that case several jobs could be used even if `None` is passed to `Parallel` for `n_jobs`. I'm not sure how to get this information from joblib's public API, it seems to be stored in the private thread-local variable `joblib.parallel._backend`.

note the transformer would still work, and some parallelization would still happen (across different transformers), just the splitting of GAP and MinHash wouldn't happen, so maybe this isn't a big deal if users don't use `parallel_config` very often

Reply:
Also I'm not sure why this is a property

Reply:
`_is_parallelized` does not exist anymore now so this comment doesn't apply